.github/workflows/volume_group_operator_ci.yaml

```diff
@@ -0,0 +1,98 @@
+        default: false
+        type: boolean
+        required: false
+env:
+  IMAGE_VERSION: '1.11.0'
+jobs:
+  prepare_env: 
+    name: "Prepare env"
+    runs-on: ubuntu-latest
+    env:
+      PRODUCTION: "${{ inputs.production }}"
+      BUILD_NUMBER: "${{ github.run_number }}"
+      PROD_REPOSITORY: volume-group-operator
+      STAGING_REPOSITORY: volume-group-operator-staging
+    steps:
+       - name: Checkout code
+         uses: actions/checkout@v3.2.0
+       - uses: FranzDiebold/github-env-vars-action@v2.7.0
+       - name: Setup dependencies
+         id: setup_dependencies
+         run: |
+           build/ci/github_actions/setup_ci_dependencies.sh
+    outputs:
+      repository: "${{ steps.setup_dependencies.outputs.repository }}"
+      specific_tag: "${{ steps.setup_dependencies.outputs.specific_tag }}"
+      global_tag: "${{ steps.setup_dependencies.outputs.global_tag }}"
+
+  build_and_push_images:
+    name: "Build and push images"
+    runs-on: ubuntu-latest
+    env:
+      REGISTRY: quay.io/csiblock
+    needs:
+      - prepare_env
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3.2.0
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: |
+            ${{ env.REGISTRY }}/${{ needs.prepare_env.outputs.repository }}
+          tags: |
+            type=raw,value=${{ needs.prepare_env.outputs.global_tag }}
+            type=raw,value=${{ needs.prepare_env.outputs.specific_tag }}
+          labels: |
+            commit=${{ github.sha }}
+      - name: Login to quay.io
+        uses: docker/login-action@v2
+        with:
+          registry: quay.io
+          username: '${{ secrets.CSIBLOCK_DOCKER_REGISTRY_USERNAME }}'
+          password: '${{ secrets.CSIBLOCK_DOCKER_REGISTRY_PASSWORD }}'
+      - name: Build and push volume-group-operator
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          platforms: linux/amd64,linux/ppc64le,linux/s390x
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+          file: Dockerfile
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+          build-args: |
+            VERSION: ${{ env.IMAGE_VERSION }}
+            BUILD_NUMBER: ${{ github.run_number }}
+      - name: save image names
+        run: |
+          echo ${{ env.REGISTRY }}/${{ needs.prepare_env.outputs.repository }} >> /tmp/images.txt
+          echo ${{ needs.prepare_env.outputs.specific_tag }} >> /tmp/images.txt
+          echo ${{ needs.prepare_env.outputs.global_tag }} >> /tmp/images.txt
+      - uses: actions/upload-artifact@v3
+        with:
+          name: images
+          path: /tmp/images.txt
```

Comment:
Why did you put the txt file in /tmp folder?
can it be saved/printed in the CI job as well?

Reply:
we upload it as an artifact like here:
![image](https://user-images.githubusercontent.com/45543087/209929502-a30d79e2-9313-46b9-aed7-03323173d571.png)

Reply:
the /tmp folder is for putting it somewhere in the github actions VM that it could upload it

Reply:
do you want me to add a stage that all it does is printing the images there, it's easy to do

Reply:
sounds good, that way we will have one known place to see all the info regarding those images.

Reply:
I not sure I understood, so you want me to add the step or not?

Reply:
yes please

Reply:
done